src/pages/App/index.js

```diff
@@ -43,33 +41,66 @@ class App extends React.Component {
         const tokenResult = await user.getIdTokenResult();
         const { claims } = tokenResult;
 
-        ClaimsSingleton.setClaims(claims);
         this.setState({
-          authUserClaims: Object.keys(claims),
+          userClaims: {
+            user_id: claims.user_id,
```

Comment:
userID

Reply:
How about userId?

Reply:
https://airbnb.io/javascript/#naming--Acronyms-and-Initialisms

we just need to pick one

Reply:
but no snake case